src/main/java/core/basesyntax/FileEditor.java

```diff
@@ -0,0 +1,233 @@
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.List;
+import java.util.Scanner;
+
+public class FileEditor {
+    Scanner scanner = new Scanner(System.in);
+
+    public void startWork() {
+        System.out.println("This is my small console app for creating/editing text files\n"
+                + "Here is the list of available operations:\n"
+                + "create path filename     will create a new file in path directory\n"
+                + "read path filename       will read file from path directory\n"
+                + "info path filename       will show you a short brief about file\n"
+                + "help                     will show all available commands\n"
+                + "help command             will tell you a bit more about chosen command\n"
+                + "exit                     will finish this program\n\n"
+                + "Type your command to start working with app\n"
+                + "IMPORTANT: you don't need to type .txt at the end of the filename\n");
+
+        String input = scanner.nextLine();
+        String[] inputWords = input.split(" ");
+
+        if (inputWords.length == 1) {
+            switch (inputWords[0].toLowerCase()) {
+                case ("help") :
+                    help();
+                    break;
+                case ("exit") :
+                    System.out.println("Thank you for using my app!\n"
+                            + "Have a nice day!");
+                    break;
+                default :
+                    writeTextToFile(input);
+            }
+        } else if (inputWords.length == 2) {
+            if (inputWords[0].toLowerCase().equals("help")) {
+                commandHelp(inputWords[1]);
+            } else {
+                writeTextToFile(input);
+            }
+        } else if (inputWords.length == 3) {
+            switch (inputWords[0].toLowerCase()) {
+                case ("create") :
+                    createFile(inputWords[1],inputWords[2]);
+                    break;
+                case ("read") :
+                    readFile(inputWords[1], inputWords[2]);
+                    break;
+                case ("info") :
+                    fileInfo(inputWords[1], inputWords[2]);
+                    break;
+                default :
+                    writeTextToFile(input);
+                    break;
+            }
+        } else {
+            writeTextToFile(input);
+        }
+
+    }
+
+    private void createFile(String filepath, String filename) {
+        Path directoryPath = Paths.get(filepath);
+        Path filePath = Paths.get(filepath + File.separator + filename + ".txt");
+        if (!Files.exists(directoryPath)) {
+            System.out.println("This directory does not exist. Create this directory? (Y/N)");
+            String answer = scanner.nextLine();
+            if (answer.toLowerCase().equals("y")) {
```

Comment:
Can we move this part of the functionality out of this method? For example, if the file directory does not exist - throw an exception and catch it in the next layer of our app. Then just call the method to create a directory and call method `createFile` again?